README.md

```diff
@@ -2,6 +2,11 @@
 
 # Kubeapps Installer
 
+Kubeapps Installer is a [Bitnami](https://bitnami.com) tool to super-charge your Kubernetes cluster with:
+ * Your own app [dashboard](https://kubeapps.com/), allowing you to deploy Kubernetes-ready apps into your cluster with a single click.
+ * [Kubeless](http://kubeless.io/) - a Kubernetes-native Serverless Framework, compatible with [serverless.com](https://serverless.com).
+ * [SealedSecrets](https://github.com/bitnami/sealed-secrets) - a SealedSecret can be decrypted only by the controller running in the cluster and nobody else (not even the original author).
+
```

Comment:
Shall we mention `nginx-controller` as a component of kubeapps or it's just a supplementary?

Reply:
actually I am going to land the new manifest including the nginx-controller deployment so I just ask here although there is no ingress-controller in the master branch atm

Reply:
well, are we installing it because charts and kubeless needs it or as a default ingress controller to populate the cluster. I understand that from a technical point of view it doesnt make a difference, but if we are installing primarily to support charts and kubeless I wouldn't mention it.